webapp/store/app/hooks/index.js

```diff
@@ -45,3 +45,19 @@ export const useRegions = () => {
 
   return regions
 }
+
+export const sortRegionGroups = (regionGroups, i18n) => {
+  return Object.fromEntries(Object.entries(regionGroups).map(([key, regions]) => [key, sortRegions(regions, i18n)]))
+}
+
+export const useRegionGroups = () => {
+  const i18n = useI18n()
+  const dispatch = useDispatch()
+  const regionGroups = useSelector(AppState.getRegionGroups)
+
+  useOnUpdate(() => {
+    dispatch(AppActions.updateRegionGroups(sortRegionGroups(regionGroups, i18n)))
```

Comment:
region groups must be ordered by 'order' not by label

Reply:
This only sorts the array in the result set, does not change the order of the groups